daru.gemspec

```diff
@@ -69,7 +69,7 @@ EOF
   spec.add_development_dependency 'dbi'
   spec.add_development_dependency 'activerecord', '~> 4.0'
   spec.add_development_dependency 'sqlite3'
-  spec.add_development_dependency 'rubocop', '>= 0.40.0'
+  spec.add_development_dependency 'rubocop', '~> 0.49.0'
```

Comment:
Never checked that, to be honest :)
And, in fact, in my own gems I am lately prefer `Gemfile`